web/src/routes/Page1.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { ReactElement } from 'react'
+import PageErrorBoundary from 'src/components/PageErrorBoundary'
+
+export default (): ReactElement<{}> => {
```

Comment:
Is that a common return type rule? I was using JSX.Element

Reply:
@vladyslavvolkov I'm not sure what's a common type. You can always look them up. JSX.Element is actually worse and just means ReactElement<any, any>; 

ReactElement<{}> isn't much better ... I'm not sure I ever needed a return type for components ... but i hope it'll catch a potential bug one day